packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/all-corejs-3.20/output.mjs

```diff
@@ -0,0 +1,57 @@
+var _context, _context2, _context3, _context4;
+
+import _forEachInstanceProperty from "core-js-pure/stable/instance/for-each.js";
+import _mapInstanceProperty from "core-js-pure/stable/instance/map.js";
+import _filterInstanceProperty from "core-js-pure/stable/instance/filter.js";
+import _Map from "core-js-pure/stable/map/index.js";
+import _dotAllInstanceProperty from "core-js-pure/stable/instance/dot-all.js";
+import _stickyInstanceProperty from "core-js-pure/stable/instance/sticky.js";
+import _Symbol from "core-js-pure/stable/symbol/index.js";
+import _Symbol$matchAll from "core-js-pure/stable/symbol/match-all.js";
+import _replaceAllInstanceProperty from "core-js-pure/stable/instance/replace-all.js";
+import _URL from "core-js-pure/stable/url/index.js";
```

Comment:
This entry and entries below are invalid, but this bug is out of the scope of this PR.